Python/Product/PythonTools/PythonTools/Refactoring/LocationPreviewItem.cs

```diff
@@ -30,66 +32,128 @@ namespace Microsoft.PythonTools.Refactoring {
     class LocationPreviewItem : IPreviewItem {
         private readonly FilePreviewItem _parent;
         private readonly string _text;
-        private readonly VariableType _type;
-        private readonly int _lineNo, _columnNo;
         private Span _span;
         private bool _checked = true;
         private static readonly char[] _whitespace = new[] { ' ', '\t', '\f' };
 
         public LocationPreviewItem(VsProjectAnalyzer analyzer, FilePreviewItem parent, AnalysisLocation locationInfo, VariableType type) {
-            _lineNo = locationInfo.Line;
-            _columnNo = locationInfo.Column;            
+            Debug.Assert(locationInfo.Column >= 1, "Invalid location info (Column)");
+            Debug.Assert(locationInfo.Line >= 1, "Invalid location info (Line)");
+            Line = locationInfo.Line;
+            //Column = locationInfo.Column;
             _parent = parent;
```

Comment:
If we return before Column is assigned, will that cause trouble?  It will be 0, and it's supposed to be 1-based.  For example, DisplayPreview below will end up setting a start index of -1.